crates/hc_sandbox/Cargo.toml

```diff
@@ -47,13 +47,13 @@ holochain_util = { version = "^0.5.0-dev.1", path = "../holochain_util", feature
 ] }
 holochain_nonce = { version = "^0.5.0-dev.2", path = "../holochain_nonce" }
 holochain_trace = { version = "^0.5.0-dev.1", path = "../holochain_trace" }
-kitsune2_api = "=0.0.1-alpha.17"
-kitsune2_core = "=0.0.1-alpha.17"
+kitsune2_api = "0.1.0"
+kitsune2_core = "0.1.0"
 nanoid = "0.4"
 serde = { version = "1.0", features = ["derive"] }
 serde_yaml = "0.9"
 serde_json = "1.0"
-sodoken = "=0.0.11"
+sodoken = "=0.0.904-alpha"
```

Comment:
Do we want to bump this to a proper version?

Reply:
Yes. With the tests passing here in holochain, let's bump sodoken to `0.1.0`.